tardis/conftest.py

```diff
@@ -56,6 +56,11 @@ def pytest_addoption(parser):
 
 
 def pytest_configure(config):
```

Comment:
@yeganer No it doesn't, though it is true that `pytest_configure` in higher level directories will be called first, followed by any `pytest_configure`s in child directories. I don't think there are any conflicts and it is safe to move it in. Should I move integration tests config and dokuwiki connection fixture as well ? (I will be rewriting history in this case to avoid mess :sweat_smile: )